tests/UI/pages/commonPage.js

```diff
@@ -341,4 +341,20 @@ module.exports = class CommonPage {
   uppercaseFirstCharacter(word) {
     return `${word[0].toUpperCase()}${word.slice(1)}`;
   }
+
+  /**
+   * Get a float price from text
+   * @param page
+   * @param selector
+   * @param timeout
+   * @returns {Promise<number>}
+   */
+  async getPriceFromText(page, selector, timeout = 0) {
+    await page.waitForTimeout(timeout);
+    const text = await this.getTextContent(page, selector);
+
+    const number = Number(text.replace(/[^0-9.-]+/g, ''));
```

Comment:
Are you sure it's working ? The `.` character in a regex is a special one, shouldn't it be escaped ?
Poke @matks @Progi1984

Reply:
I'm sure it's working, it has been working for quite a while actually, I just moved it to the parent class.

Reply:
In theory dots must be escaped inside regexps (so `\.`) but since it's inside brackets (which is a "container" delimiter) maybe it's not needed 🤔  I checked in php it works too

Reply:
OK so if it's working, it's good ! The brackets are the only difference from the situation where I would be confident in saying it doesn't work :laughing:

Reply:
I'm very confident about this, it was done by @PierreRambaud :laughing: